src/jaxsim/rbda/actuation/common.py

```diff
@@ -0,0 +1,12 @@
+import jaxsim.typing as jtp
+from jaxsim.utils import JaxsimDataclass
+
+
+class ActuationParams(JaxsimDataclass):
+    """
+    Parameters class for the actuation model.
+    """
+
+    τ_max: jtp.Float = 3000.0  # Max torque (Nm)
+    ω_th: jtp.Float = 30.0  # Threshold speed (rad/s)
+    ω_max: jtp.Float = 100.0  # Max speed for torque drop-off (rad/s)
```

Comment:
I'm usually a fan of unicode chars, but I'm super convinced of using them with user-exposed variables, e.g. in IPython these attribute would be difficult to access/create. Can we switch to ascii chars?

```suggestion
    max_torque: jtp.Float = 3000.0  # (Nm)
    threshold_speed: jtp.Float = 30.0  # (rad/s)
    dropoff_speed: jtp.Float = 100.0  # (rad/s)
```

Reply:
Done in ba556cd4e97d43b10710ad06c9b3af842ddbcd8b